assets/blocks/course-outline/index.js

```diff
@@ -1,4 +1,10 @@
-import './course-outline-block';
+import { registerBlockType } from '@wordpress/blocks';
+import CourseOutlineBlock from './course-block';
 import './module-block';
 import './lesson-block';
 import './store';
+
+[ CourseOutlineBlock ].forEach( ( block ) => {
```

Comment:
I think the idea would be to make it for all the blocks, right?

Reply:
Yup, I'll add them in a new PR